compiler/rustc_ast/src/tokenstream.rs

```diff
@@ -446,14 +446,18 @@ impl TokenStream {
         self.clone().into_trees()
     }
 
+    pub fn trees_ref(&self) -> CursorRef<'_> {
```

Comment:
```suggestion
    pub fn trees(&self) -> CursorRef<'_> {
```
This function can be made the default.